packages/glibc.rb

```diff
@@ -3,20 +3,21 @@
 class Glibc < Package
   description 'The GNU C Library project provides the core libraries for GNU/Linux systems.'
   homepage 'https://www.gnu.org/software/libc/'
-  version LIBC_VERSION
   license 'LGPL-2.1+, BSD, HPND, ISC, inner-net, rc, and PCRE'
-  compatibility 'all'
 
   depends_on 'gawk' => :build
   depends_on 'libgd' => :build
   depends_on 'libidn2' => :build
   depends_on 'texinfo' => :build
+  depends_on 'hashpipe' => :build
 
   # enable-obsolete-rpc should NOT be included as a build option as
   # this functionality is provided by libtirpc
 
   case LIBC_VERSION
   when '2.23'
+    version LIBC_VERSION
+    compatibility 'i686'
```

Comment:
These lines don't need to be here.

Reply:
Versioning should be per Glibc version, no?

Otherwise Glibc 2.27 users will get update suggestions when we update Glibc 2.32...

Reply:
That's what I'm saying.  `version LIBC_VERSION` can be set at the top.  The way I envision it working is `LIBC_VERSION` is what drives the decision and not compatibility.  That can be set at the top to all.